### ISSUE TITLE
Application installation concept

### ISSUE THREAD

**Issue by daniel-vera-g** [daniel-vera-g/dotfiles#222]
- Develop better concept to automatically install new applications
- Use Ansible?
- Integrate in testing concept

---

Applications installation:
	Apt:
		autokey
		vim
		terminator
		expect
		solar
		system-load-indicator
		zathura
		code
		docker
		git-flow
		safeeyes
		virtualbox
		pandoc
		xclip
		powerline
		tmux
		httpie
		htop
		texlive
		fasd
		brave
		tree
		okular
		shutter
		zsh
		ctags
		exfat
		curl
		borgbackup
		lynx
		copyq
		redshift
		tweak-tool
		caffeine
		mpv
		guake
		spotify
		gimp
		lazygit
		mysql-client	
	dpkg
		nnn
		fd
		rg
		bat
		vagrant
	npm
		pomd
		eslint
		commitzen
		node
		emoj
		spaceship-theme
	pip
		gitup
		nvim
		thefuck
		glances
		mycli
	snap
		qalculate
		inkscape
	web-vcs
		language-tool
		yEd
		mgitstatus
		has
		ctop
		git-open
		firefox-developer
		navi
		marker
		tldr
		firacode
		lazydocker
		exa
		hub
		ghi
		pandoc-eisvogel
		protonvpn
